### DISCUSSION TITLE
Remove Github Student Pack Benefits

### DISCUSSION THREAD

**Discussion by IAMLEIIKUN** [community/community#43060] [GitHub Education]
I'm going straight to the point so my edu email was hacked and it was used to an another account that it wasn't mine. I tried to logged in but it said that I can't login with that email cause it's only use for a billing organization something like that. 

I really wanted to remove the benefits on that hacker account by removing my edu mail inside it. I still have the access of my email edu account tho cause I message my school to change it password yet the benefits are still there.

I tried to message github via email but it only replied me to post a help here, hopefully someone can assist me. Thank you very much and have a good day and new year!

**Comment by drewsix9** [comment]
Same here, my account got hacked (still can access my account) thus github marking it as spam account. I can no longer use third party applications because of it. My problem still not solved
